docs/user-guide/src/bmo/introduction.md

```diff
@@ -0,0 +1,87 @@
+<h1>Bare Metal Operator</h1>
+
+The Bare Metal Operator (BMO) is a custom Kubernetes controller that deploys baremetal hosts, represented in Kubernetes by BareMetalHost (BMH), as Kubernetes nodes. To this end Ironic is used.
+
+The BareMetalHost represents a bare metal host (server). The BareMetalHost contains information about the server as shown below. For brevity, some part of the output are omitted, but we can classify the fields into the following broad categories.
+
+1. known server properties: Fields such as `bootMACAddress` properties of the server and are known in advance.
+2. unknown server properties: Fields such as `CPU` and `disk` are properties of the server and are discovered by Ironic.
+3. user supplied: Fields such as `image` are supplied by user to dictate boot image for the server.
+4. dynamic fields: Fields such as IP could be dynamically assigned to the server at run time by DHCP server.
+
+During the life cycle of a bare metal host (server), upgrade is one example, some of these fields change with information coming from Ironic or other controllers while fields, such as MAC address, do not change.
```

Comment:
```suggestion
During the life cycle of a bare metal host, upgrade is one example, some of these fields change with information coming from Ironic or other controllers while fields, such as MAC address, do not change.
```
There is one like this already before this, perhaps no need to repeat it again

Reply:
done